saleor/graphql/order/mutations/orders.py

```diff
@@ -424,29 +424,29 @@ class OrderUpdateMeta(UpdateMetaBaseMutation):
     class Meta:
         description = "Updates meta for order."
         model = models.Order
-        permissions = "orders.manage_orders"
+        permissions = ("order.manage_orders",)
```

Comment:
@salwator  Should we protect order public meta? Personally, I think that PWA client should be able to store some data dedicated to order.

Reply:
And the second question. Should we use the order's ID here instead of `order.token`? 
cc @maarcingebala

Reply:
I assume this also apply to clearing public meta? @salwator @korycins